Assets/Scripts/Notebook/Notebook Interactions/Counter.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using Anivision.PlayerInteraction;
+
+namespace Anivision.NotebookSystem
+{
+    public class Counter : MonoBehaviour
+    {
+        public TextMeshPro countTMP;
+        public HeadsetCollide headsetCollide;
+
+        private Chapter _chapter;
+        private Grabber[] _Grabbers;
+        private Grabber _RGrabber;
+        private Grabber _LGrabber;
+        private int _currCount;
+        private string _countString;
+
+        private void OnEnable()
+        {
+            _chapter = gameObject.GetComponent<Chapter>();
+            if (_chapter == null) UnityEngine.Debug.LogError("Trying to access the chapter of this gameobject, but there is none.");
+
+            _Grabbers = FindObjectsOfType<Grabber>();
+            if (_Grabbers.Length > 0)
+            {
+                _RGrabber = _Grabbers[0];
+
+                if (_Grabbers.Length > 1)
+                {
+                    _LGrabber = _Grabbers[1];
+                }
+            }
+
+            headsetCollide.onCollide.AddListener(Eat);
+            _currCount = 0;
```

Comment:
So everytime we enable the game object that this script is attached to, the count gets reset? Does that mean that when you switch to vision switch page, the count also gets reset?